app/frontend/src/components/CommunityGuidelines/CommunityGuidelines.tsx

```diff
@@ -83,22 +104,19 @@ export default function CommunityGuidelines({
         {error && <Alert severity="error">{error}</Alert>}
```

Comment:
This is also redundant since the error gets thrown early above, so this never gets displayed.
```suggestion
```